ios/Integreat/Info.plist

```diff
@@ -63,5 +63,6 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>NSLocationWhenInUseUsageDescription</key>
```

Comment:
The value is missing here: https://developer.apple.com/documentation/bundleresources/information_property_list/nslocationwheninuseusagedescription

I can do this on mac tough

Reply:
The strings are located in InfoPlist.strings, I don't think that it's necessary to put it in Info.plist as well.